### ISSUE TITLE
mobile/ enforce sufficient color contrast

### ISSUE THREAD

**Issue by serdarbahar** [bounswe/bounswe2025group5#564]
### Task Description
Sufficient color contrast throughout the application should be present according to the accessibility requirements. This issue will deal with implementing the feature in the mobile application. The relevant [requirement](https://github.com/bounswe/bounswe2025group5/wiki/Requirements-of-CMPE451) is 2.1.2.6.

### Why Is This Needed?
- To follow accessibility guidelines throughout our project

### Acceptance Criteria
- [ ] minimum color contrast ratio of 4.5:1 for text 3:1 for large text/UI is implemented

### Deadline
20.11.2025 - 23:59
